docker-compose.override.yml.dist

```diff
@@ -43,7 +43,16 @@ services:
             - 127.0.0.1:9000:9000
         environment:
             CICD_CORS_ALLOW_ALL_ORIGINS: "false"
-            CICD_CORS_ALLOW_ORIGINS: "http://localhost:4200"
+            CICD_CORS_ALLOW_ORIGINS:     "http://localhost:4200"
+            CICD_EXTERNAL_URL:           https://my-cicd.serveo.net
```

Comment:
Why do we need an external URL in the local environment?

Reply:
As for PKE for example, the external system that we are programming tries to call back the programming entity, in this case, GitLab checks if the registered WebHook URL is callable, GitHub does it as well, but in case of GitLab if the check fails the webhook registration fails as well.

Reply:
I see, thanks for clarification.